app/lib/katello/validators/content_view_erratum_filter_rule_validator.rb

```diff
@@ -2,9 +2,9 @@ module Katello
   module Validators
     class ContentViewErratumFilterRuleValidator < ActiveModel::Validator
       def validate(record)
-        if record.errata_id.blank? && record.start_date.blank? && record.end_date.blank? && record.types.blank?
+        if record.errata_id.blank? && record.start_date.blank? && record.end_date.blank? && record.types.blank? && record.allow_other_types == false
           invalid_parameters = _("Invalid erratum filter rule specified, Must specify at least one of the following:" \
-                                     " 'errata_id', 'start_date', 'end_date' or 'types'")
+            " 'errata_id', 'start_date', 'end_date', 'types', or 'allow_other_types'")
           record.errors.add(:base, invalid_parameters)
           return
```

Comment:
I am also seeing an error updating a date range filter by removing the regular types and keeping only the "Other" type selected : 
`May not add an id rule to a filter that has an existing type or date range rule.`

Reply:
I'll look into this shortly. Thank you Samir!

Reply:
Samir, I haven't been able to replicate this on my end. How can I reproduce this?

Reply:
I hit a similar weird issue:
1) Create errata date filter
2) Set the types to only be other
3) Add a date range
4) Save the filter rule
5) Go back into the filter rule
6) See UI error and blank screen
7) Refresh the page
8) See that the filter rule was somehow turned from an errata date rule to an errata by ID rule